core/vm/evm.go

```
@@ -130,6 +130,10 @@ type EVM struct {
 
 	readOnly   bool   // Whether to throw on stateful modifications
 	returnData []byte // Last CALL's return data for subsequent reuse
+
+	// EIP-8082 - Contract Event Subscription
+	SubscriptionManager *SubscriptionManager      // Manages event subscriptions
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(goimports)